.gitignore

```diff
@@ -1,11 +1,3 @@
-node_modules
```

Comment:
Why are you removing all these from the ignore file?

Reply:
If you check the rest of the `.gitignore` file, you'll see that `node_modules/` is _already_ listed (in the default location for `.gitignore`). There is no need for this duplicate.